src/pages/Appointments/components/MultiPractitionerSelect.tsx

```
@@ -0,0 +1,221 @@
+  const getItemValue = (user: UserBase) => {
+    return formatName(user) + " " + user.id;
+  };
+
+  return (
+    <div className="flex items-center gap-2">
+      <div className="order-last sm:order-first">
+        {selected && selected.length > 0 && (
+          <div className="flex items-center gap-1">
+            {selected.map((user) => (
+              <>
+                <TooltipProvider>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)